src/main/java/org/adridadou/ethereum/propeller/rpc/EthereumRpc.java

```diff
@@ -87,7 +87,8 @@ public EthHash submit(TransactionRequest request, Nonce nonce) {
         UInt256 nonceInt = UInt256.valueOf(nonce.getValue());
         Wei gasPriceWei = Wei.valueOf(gasPrice.getPrice().inWei());
         Gas gasLimitWei = Gas.valueOf(request.getGasLimit().getUsage());
-        Address address = Address.fromBytes(Bytes.of(request.getAddress().toData().data));
+        Address address = null;
+        //Address address = Address.fromBytes(Bytes.of(request.getAddress().toData().data));
```

Comment:
great that it works! But remember to put an if here for a general solution.

I.e. if(request.getAddress().equals(EthAddress.empty()) null else Address.fromBytes(request.getAddress.toData.data) (something like that)

Reply:
just pushed that after testing again it worked with Kaleido on OL backend